src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController+Extension.swift

```diff
@@ -19,6 +19,24 @@
 
 import UIKit
 
+
+enum OnboardingPageType: Int, CaseIterable {
```

Comment:
I think it makes more sense to move this to a separate file instead of this extension collection

Reply:
OK, new file for enum